lld/test/ELF/gnu-lto/hello.test

```diff
@@ -0,0 +1,30 @@
+// REQUIRES: gnu_lto, gcc
+// Test if we can link a simple LTO'ed program.
+// Ensure that LLD was used by checking for the output from LLD_VERSION.
+
+// RUN: gcc -c -flto -x c %s -o %t.dynamic.o
+// RUN: gcc -B%llvm_obj_root -fuse-ld=lld -flto %t.dynamic.o -o %t.dynamic
+// RUN: llvm-objdump -t -j .comment -s %t.dynamic | \
+// RUN:   FileCheck --check-prefix=OBJDUMP %s
+// RUN: llvm-readelf -s %t.dynamic | \
+// RUN:   FileCheck --check-prefix=READELF %s
+
+// RUN: gcc -c -flto -x c -fPIE %s -o %t.pie.o
+// RUN: gcc -B%llvm_obj_root -fuse-ld=lld -flto -pie %t.pie.o -o %t.pie
+// RUN: llvm-objdump -t -j .comment -s %t.pie \
+// RUN:   | FileCheck --check-prefix=OBJDUMP %s
+// RUN: llvm-readelf -s %t.pie | \
+// RUN:   FileCheck --check-prefix=READELF %s
+
+#include <stdio.h>
```

Comment:
I recommend removing any dependency on the C-library. If we don't need the output to run, just that the link succeeded then 
```int f(void) { return 0; }```
Would suffice. 

This would permit the test to work with just GCC being able to compile a file, but without needing all the include and library paths configured.